File: CS/WinWebSolution.Module/Updater.cs
using System;
using DevExpress.Xpo;
using DevExpress.ExpressApp.Updating;

namespace WinWebSolution.Module {
    public class Updater : ModuleUpdater {
        public Updater(Session session, Version currentDBVersion) : base(session, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
            using (UnitOfWork uow = new UnitOfWork(Session.DataLayer)) {
                Product product = new Product(uow);
                product.Name = "Chai";
                for (int i = 0; i < 10; i++) {
                    Order order = new Order(uow);
                    order.Product = product;
                    order.Description = "Order " + i.ToString();
                    order.Total = i;
                }
                uow.CommitChanges();
            }
        }
    }
}

File: VB/WinWebSolution.Module/Updater.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Xpo
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Updating

Namespace WinWebSolution.Module
	Public Class Updater
		Inherits ModuleUpdater
		Public Sub New(ByVal objectSpace As IObjectSpace, ByVal currentDBVersion As Version)
			MyBase.New(objectSpace, currentDBVersion)
		End Sub
		Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
			MyBase.UpdateDatabaseAfterUpdateSchema()
			
				Dim product As Product = ObjectSpace.CreateObject(Of Product)()
				product.Name = "Chai"
				For i As Integer = 0 To 9
					Dim order As Order = ObjectSpace.CreateObject(Of Order)()
					order.Product = product
					order.Description = "Order " & i.ToString()
					order.Total = i
				Next i
				ObjectSpace.CommitChanges()
			
		End Sub
	End Class
End Namespace